modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/chapters/ChaptersPage.kt

```diff
@@ -30,14 +37,26 @@ fun ChaptersPage(
         modifier = modifier
             .background(backgroundColor)
             .fillMaxSize()
-            .padding(top = 16.dp),
+            .padding(top = if (FeatureFlag.isEnabled(Feature.DESELECT_CHAPTERS)) 0.dp else 16.dp),
     ) {
+        if (FeatureFlag.isEnabled(Feature.DESELECT_CHAPTERS)) {
+            item {
+                ChaptersHeader(
+                    totalChaptersCount = totalChaptersCount,
+                    hiddenChaptersCount = totalChaptersCount - chapters.filter { it.chapter.selected }.size,
+                    onSkipChaptersClick = onSkipChaptersClick,
+                    isTogglingChapters = isTogglingChapters,
+                )
+            }
+        }
         itemsIndexed(chapters, key = { _, state -> state.chapter.index }) { index, state ->
             ChapterRow(
                 state = state,
+                isTogglingChapters = isTogglingChapters,
                 onSelectionChange = onSelectionChange,
                 onClick = { onChapterClick(state.chapter, state is ChaptersViewModel.ChapterState.Playing) },
                 onUrlClick = { onUrlClick(state.chapter.url.toString()) },
+                modifier = Modifier.animateItemPlacement(),
```

Comment:
`animateItemPlacement()` is an experimental modifier property. I've tested it on API 24 emulator.